src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -276,25 +276,86 @@ private static void handleKeyValueConfig(TestInputConfiguration.Builder testInpu
     private static String getFullyQualifiedClassName(String filePath, String moduleName)
             throws CheckstyleException {
         String fullyQualifiedClassName;
-        if (moduleName.startsWith("com.")) {
-            fullyQualifiedClassName = moduleName;
+        final String resolvedModuleName = resolveModuleName(moduleName);
+        if (resolvedModuleName.startsWith("com.")) {
+            fullyQualifiedClassName = resolvedModuleName;
         }
         else {
             final String path = SLASH_PATTERN.matcher(filePath).replaceAll("\\.");
-            final int endIndex = path.lastIndexOf(moduleName.toLowerCase(Locale.ROOT));
+            final int endIndex = path.lastIndexOf(resolvedModuleName.toLowerCase(Locale.ROOT));
             if (endIndex == -1) {
-                throw new CheckstyleException("Unable to resolve module name: " + moduleName
+                throw new CheckstyleException("Unable to resolve module name: " + resolvedModuleName
                 + ". Please check for spelling errors or specify fully qualified class name.");
             }
             final int beginIndex = path.indexOf("com.puppycrawl");
-            fullyQualifiedClassName = path.substring(beginIndex, endIndex) + moduleName;
+            fullyQualifiedClassName = path.substring(beginIndex, endIndex) + resolvedModuleName;
             if (!fullyQualifiedClassName.endsWith("Filter")) {
                 fullyQualifiedClassName += "Check";
             }
         }
         return fullyQualifiedClassName;
     }
 
+    private static String resolveModuleName(String simpleName) {
+        String resolvedName = simpleName;
```

Comment:
By this method you try to do what maun code already know how to do.
Better to reuse main code.

To speedup this PR acceptance, I recommend to simply hardcoded map of few modules that you have in your examples to their fully qualified names.